Gruntfile.js

```diff
@@ -0,0 +1,59 @@
+/*
+
+This Grunt script sets up a local dev server and enables live reload when source files are 
+changed.
+
+*/
+
+module.exports = function(grunt) {
+
+  grunt.loadNpmTasks("grunt-contrib-connect");
+  grunt.loadNpmTasks("grunt-contrib-watch");
+  
+  grunt.initConfig({
+    connect: {
+      dev: {
+        options: {
+          livereload: true
+        }
+      }
+    },
+    watch: {
+      options: {
+        livereload: true
+      },
+      html: {
+        files: "*.html",
+        tasks: ["nothing"]
+      },
+      js: {
+        files: ["js/*.js"],
+        tasks: ["browserify"]
+      }
+    }
+  });
+
+  //task to trigger reload
+  grunt.registerTask("nothing", function() {});
+  
+  //default task
+  grunt.registerTask("default", ["browserify", "connect:dev", "watch"]);
```

Comment:
By default, the Grunt task builds your scripts, but it also starts a development server on port 8000, and watches files so that when they change, it will rebuild them and even reload your browser for you. This cuts down on development time by a fairly significant amount, especially if you can just keep the browser on a second monitor.